### PR TITLE
Pass relevant ENV VARs to application container

### PR DESCRIPTION
**Pull request by rhcarvalho**
@bparees @mnagy @mfojtik PTAL

### CODE REVIEW

File: openshift/templates/nodejs-mongodb.json [hunk 1]
```diff
@@ -160,6 +160,10 @@
                 ],
                 "env": [
                   {
+                    "name": "DATABASE_SERVICE_NAME",
```
Comment (rhcarvalho) [inline openshift/templates/nodejs-mongodb.json]: We need this to be able to read the env var `${DATABASE_SERVICE_NAME}_SERVICE_HOST` and `${DATABASE_SERVICE_NAME}_SERVICE_PORT`.

File: openshift/templates/nodejs.json [hunk 2]
```diff
@@ -157,6 +157,28 @@
                   {
                     "containerPort": 8080
                   }
+                ],
+                "env": [
```
Comment (rhcarvalho) [inline openshift/templates/nodejs.json]: We need this to be able to connect to an existing MongoDB. We have already the parameters defined below in the template, and you can set them via Web Console, but then without this patch they are not passed to the application container.

**Comment by mfojtik** [comment]
LGTM

**Comment by mnagy** [comment]
Will test in a second.

**Comment by mnagy** [comment]
LGTM, tests work :+1:

**Pull request merged**